python/apps/in-mem-dynamic.py

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Couple things here:

1: `fire` is a way better library than `argparse` to make things simple and runnable.  I'll update these later to use it, but I'm a huge fan in how easy it makes stuff!
2:  Eventually I'd like to move these out to like `examples/python/` in the top level, and we can throw in some jupyter notebooks too.  it doens't need to go there right now, just wanted to comment on my intent for these kinds of py files :)